<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCoinsEntry</class>
 <widget class="QFrame" name="SendCoinsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pay &amp;To:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>payTo</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="btnSubject">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/add+</normaloff>:/icons/add+</iconset>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A&amp;mount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>payAmount</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="txtFrom"/>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="payToLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="payTo">
       <property name="toolTip">
        <string>The address to send the payment to  (e.g. Vvpywwpq4uKGvouhNv7jzKZ4uoYm3ECsbM)</string>
       </property>
       <property name="maxLength">
        <number>34</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addressBookButton">
       <property name="toolTip">
        <string>Choose address from address book</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="toolTip">
        <string>Remove this recipient</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="addAsLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Enter a label for this address to add it to your address book</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="BitcoinAmountField" name="payAmount"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Label:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>addAsLabel</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="lblCharRemain">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="btnFrom">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/add+</normaloff>:/icons/add+</iconset>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="hl_M" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblHMessage">
       <property name="text">
        <string>Message to Recipient option:</string>
       </property>
       <property name="indent">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout_mode">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rbt_mod_normal">
         <property name="text">
          <string>Normal</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbt_mod_vpn">
         <property name="text">
          <string>VPN Fee</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbt_mod_buydomain">
         <property name="text">
          <string>Buy Domain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbt_mod_lottery">
         <property name="text">
          <string>Encash Lottery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbt_mod_coinfreeze">
         <property name="text">
          <string>Coin Freeze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbt_mod_coinjoin">
         <property name="text">
          <string>Coin Join</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblSubject">
     <property name="text">
      <string>Subject:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblFrom">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="txtSubject"/>
   </item>
   <item row="11" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="btnMessage">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/add+</normaloff>:/icons/add+</iconset>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <layout class="QVBoxLayout" name="vLayout_Msg"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QSpinBox</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
